neptune/man/neptune_file_as_image.Rd

```diff
@@ -0,0 +1,46 @@
+\name{neptune_file_as_image}
+\alias{neptune_file_as_image}
+\title{
+  Static method for converting image objects or image-like objects to an image File value object.
+}
+\description{
+  Static method for converting image objects or image-like objects to an image File value object. This way you can upload Matplotlib figures, PIL images, NumPy arrays, as static images.
+}
+\usage{
+neptune_file_as_image(x)
+}
+\arguments{
+  \item{x}{
+An object to be converted. Supported are Altair, Bokeh, Plotly, Matplotlib interactive charts, and Pandas DataFrame objects.
```

Comment:
In docs there is `Image-like object to be converted. Supported are PyTorch tensors, TensorFlow/Keras tensors, NumPy arrays, PIL images, Matplotlib figures.`